backend/src/main/java/capstone/herbs/user/LoginData.java

```diff
@@ -8,6 +8,5 @@
         private String password;
         private String email;
         private String role;
```

Comment:
Why does `LoginData` contain a role attribute? Is it necessary for logging in?

Reply:
the users will have different role in future.

Reply:
But this shouldn't be information, that is sent to the server by the client. The server should know the role because it is stored in the database. Otherwise anybody could make himself admin by changing the request that is sent.

Reply:
I will fix it within the next pull request